resources/js/components/Monitor.vue

```diff
@@ -50,24 +55,21 @@ export default {
 
   methods: {
     postSetup: function() {
-      const api_data = this.cdash.time_chart_data;
+      const api_data = this.cdash.time_chart_data.data;
 
       // perform data marshalling before sending data to plot template
-      let formatted_data = [];
-      for (let i = 0; i < api_data.data.length; i++) {
+      const formatted_data = [];
+      for (let i = 0; i < api_data.length; i++) {
```

Comment:
That's find with me if we think that it will improve the readability in this chunk of code. In terms of other explicit `for` loops I introduced in the Timeline component, I believe that they actually make the code easier to understand. For example, when we're iterating over the sets of data and we plot each line and the circles of data points corresponding to it.